src/map/skill.cpp

```diff
@@ -11708,12 +11708,13 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 			if( !status_isdead(bl) )
 				break;
 
+			status_zap(bl, 0, tstatus->sp * (60 - 10 * skill_lv) / 100);
```

Comment:
```suggestion
			status_zap(bl, 0, tstatus->sp * 10 * skill_lv / 100);
```
Its a flat 10% per skill level.
But I think in general you have to store this calculated amount, as it is used as value for the heal.
Now you reduce it first and therefore it would result in a smaller amount of HP recovered.
Or is this intentional?